### ISSUE TITLE
unable to push the app

### ISSUE THREAD

**Issue by kulabtech** [cloudfoundry/cli#2178]
Hi Team,

We are unable to push the application to cloud foundry due to below error

Runner error: staging task with guid 832d0b8c-2f19-446d-8558-24cba11534ab failed to start",

https://user-images.githubusercontent.com/46423311/125741524-31f77c71-c390-4551-b49e-7c66a6710afd.png

Attaching the screenshot for the same in the above link. Kindly provide us a solution to this.

image

Regards,
Kulasekhar

**Comment by JenGoldstrich** [comment]
Hey @kulabtech we need a few more things to debug this issue

1. What cf-cli version are you using (run `cf -v`)
2. What sort of app are you trying to push, does this app build/start locally?
3. Can we have the manifest.yml?
4. Can you run `cf logs app-name --recent` after one of these failures, and post those logs (you might have to scrub anything private from them)

Thanks,
Jenna

**Comment by kulabtech** [comment]
thanks for the response. PFB details respectively for your questions.

1. We are using cf-cli version-7.2
2. We are running spring boot application which is being pushed from the cf-cli to IBM Cloud Foundry 
3. I have attached the manifest yml. you can refer the code in github which we have placed for your reference - https://github.com/kulabtech/PCF.git
4.  I have attached the logs in the below link. PFB link.  Kindly revert if you require any info on this issue.

[manifestfile.txt](https://github.com/cloudfoundry/cli/files/6848409/manifestfile.txt)

![image](https://user-images.githubusercontent.com/46423311/126326685-255cc1a8-8ff7-48cc-b4ce-828bb8ffb739.png)

**Comment by JenGoldstrich** [comment]
Sorry about the delay in getting back to you,

I tried to build and deploy the app you linked but on initial cf push it fails because of a missing built jar,  I tried to run maven to build this but admitadly I haven't used maven in some time so I'm not sure if I was doing it wrong but I was unable to get the app to build, if you have compilation steps, or could upload the jar I could try again.

I found this error in the [cloud controller code that talks to diego](https://github.com/cloudfoundry/cloud_controller_ng/blob/1d530dbd2194aa80d026291c67197a34d8f77e6f/lib/cloud_controller/diego/staging_completion_handler.rb#L74), this error occurs when diego (a system that maintains) failed to stage an app.

To figure this out we would need more logs I think, most likely from Diego or the Cloud Controller, those logs would be able to tell us why specifically your app is failing to stage.
